test/Search_UnitTestHelpers.hpp

```diff
@@ -72,8 +72,12 @@ auto query(ExecutionSpace const &exec_space, Tree const &tree,
            Queries const &queries)
 {
   using memory_space = typename Tree::memory_space;
+#ifdef ARBORX_ENABLE_MPI
   using value_type =
-      std::conditional_t<is_distributed<Tree>{}, Kokkos::pair<int, int>, int>;
+      std::conditional_t<is_distributed<Tree>{}, ArborX::PairIndexRank, int>;
+#else
+  using value_type = int;
+#endif
```

Comment:
Yuk

Reply:
That's why I made the `value_type` a template parameter in  #816.

Reply:
But making `value_type` a template parameter triggered a cascading change all over the place. I think it's much easier to contain that to this one place.

Reply:
I guess there are advantages to both approaches. 🙂